### PR TITLE
fix/reddit

### PR DESCRIPTION
**Pull request by steve-si**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   "name": "amp-node",
-  "version": "1.1.3",
+  "version": "1.1.4",
   "main": "Amp.js",
   "repository": "git@github.com:ScaledInference/amp-node-thin.git",
   "author": "Tao <tao@scaledinference.com>",
   "license": "MIT",
   "scripts": {
     "example": "node ./examples/example.js 3017f11895d66f82 http://amp2-dev-clientsim-0:8100",
+    "init": "gcloud compute ssh --ssh-flag=\"-L 8100:127.0.0.1:8100\"  amp2-dev-clientsim-0",
```
Comment (ajay-si) [inline package.json]: remove references to amp2-dev-xxx

File: package.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
 {
   "name": "amp-node",
-  "version": "1.1.3",
+  "version": "1.1.4",
   "main": "Amp.js",
   "repository": "git@github.com:ScaledInference/amp-node-thin.git",
   "author": "Tao <tao@scaledinference.com>",
```
Comment (ajay-si) [inline package.json]: update?

File: package.json (hunk 1 above)
Comment (steve-si) [reply package.json]: done

**Comment by yanpu-si** [comment]
Shall we keep this open until we know it is actually solving the problem?

**Comment by steve-si** [comment]
I don't think this will fix anything since they are already setting keepAlive on the http library inside of request.  Is Reddit testing with just Ajay's fix by using the bash script that Joey sent again?

**Pull request closed without merging**